### ISSUE TITLE
Update reqwest on rust

### ISSUE THREAD

**Issue by SheIITear** [curlconverter/curlconverter#230]
Hi! I tried to convert this:

```sh
curl 'https://www.mydomain.com' \
  -H 'authority: www.mydomain.com' \
  -H 'upgrade-insecure-requests: 1' \
  -H 'user-agent: Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/82.0.4085.6 Safari/537.36' \
  -H 'accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9' \
  -H 'sec-fetch-site: none' \
  -H 'sec-fetch-mode: navigate' \
  -H 'sec-fetch-user: ?1' \
  -H 'sec-fetch-dest: document' \
  -H 'accept-language: fi-FI,fi;q=0.9,en-US;q=0.8,en;q=0.7' \
  -H 'cookie: "cookie"' \
  --compressed
```

Which returned:

```rust
extern crate reqwest;
use reqwest::header;
 fn main() -> Result<(), Box<dyn std::error::Error>> {
    let mut headers = header::HeaderMap::new();
    headers.insert("authority", "www.mydomain.com".parse().unwrap());
    headers.insert("upgrade-insecure-requests", "1".parse().unwrap());
    headers.insert("user-agent", "Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/82.0.4085.6 Safari/537.36".parse().unwrap());
    headers.insert("accept", "text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9".parse().unwrap());
    headers.insert("sec-fetch-site", "none".parse().unwrap());
    headers.insert("sec-fetch-mode", "navigate".parse().unwrap());
    headers.insert("sec-fetch-user", "?1".parse().unwrap());
    headers.insert("sec-fetch-dest", "document".parse().unwrap());
    headers.insert("accept-language", "fi-FI,fi;q=0.9,en-US;q=0.8,en;q=0.7".parse().unwrap());
    headers.insert("cookie", "cookie".parse().unwrap());
    let res = reqwest::Client::new()
        .get("https://www.mydomain.com")
        .headers(headers)
        .send()?
        .text()?;
    println!("{}", res);
     Ok(())
}
```

Trying to run the rust code above using newest reqwest, returns this error: 

```
error[E0277]: the '?' operator can only be applied to values that implement 'std::ops::Try'
  --> src/main.rs:17:15
   |
17 |       let res = reqwest::Client::new()
   |  _______________^
18 | |         .get("https://www..mydomain.com")
19 | |         .headers(headers)
20 | |         .send()?
   | |________________^ the '?' operator cannot be applied to type 'impl std::future::Future'
```

Using older version of reqwest fixes this issue.

**Comment by Mariownyou** [comment]
you need to use older rewqwest version. `reqwest = "0.9.0"` works for me

**Comment by verhovsky** [comment]
Closed by #409. Now the generated code uses `::blocking`, in order to run it you need to add this to Cargo.toml:

```toml
[dependencies]
reqwest = { version = "0.11", features = ["blocking", "json"] }
```
